### PR TITLE
docs: Add Alpha Accelerator criteria

### PR DESCRIPTION
**Pull request by peasee**
## 🗣 Description

* Adds an Alpha accelerator criteria
* Updates some general formatting/references to definitions for Beta and RC

### CODE REVIEW

File: docs/criteria/accelerators/alpha.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Spice.ai OSS Data Accelerators - Alpha Release Criteria
 
-This document defines the set of criteria that is required before a data accelerator is considered to be of Alpha quality.
+This document defines the set of criteria that is required before a data Accelerator is considered to be of Alpha quality.
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]:
```suggestion
This document defines the set of criteria that is required before a Data Accelerator is considered to be of Alpha quality.
```

File: docs/criteria/accelerators/alpha.md [hunk 2]
```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
 
 ## Alpha Release Criteria
 
-TBD
+The Alpha Release Criteria for accelerators is set at a level that ensures the Accelerator operates in common conditions with a low error rate.
+
+Accelerators are intended to be more stable than Connectors, and as such the Alpha criteria is set slightly higher than that of Connectors.
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]:
```suggestion
Accelerators are intended to be more stable than Connectors, and as such the Alpha criteria is set higher than that of Connectors.
```

File: docs/criteria/accelerators/alpha.md [hunk 3]
```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
...
+
+The Alpha Release Criteria is not intended to cover any edge cases or complex functionality, so federation, streaming. However, some testing is required to cover TPC-H derived queries.
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]:
```suggestion
The Alpha Release Criteria is not intended to cover edge cases or advanced functionality, such as federation/streaming. However, testing is required to cover TPC-H derived queries.
```

File: docs/criteria/accelerators/alpha.md [hunk 4]
```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
...
+
+### All Accelerators
+
+- [ ] The accelerator implements the querying functionality of the native accelerator source in all [Access Modes](../definitions.md)
+- [ ] The accelerator supports only String, Number, Date and Binary [Core Connector Data Types](../definitions.md).
+- [ ] The accelerator executes common use case queries with a low error rate (e.g. `SELECT * FROM tbl WHERE col = 'blah'`).
+- [ ] Known [Minor and Major](../definitions.md) bugs are logged, but not required to be fixed unless needed to achieve a low query error rate and/or TPC-H success.
+  - A "low error rate" indicates that more than 90% of the time, the common use case succeeds.
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]: It's not clear to me why we would have any error rate for a given query - either it works or doesn't. Is this a load related metric or something else?

Comment (phillipleblanc) [reply docs/criteria/accelerators/alpha.md]: Or is this saying that for some hypothetic set of common use case queries, over 90% of them work?

File: docs/criteria/accelerators/alpha.md [hunk 5]
```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
...
+#### Testing
+
+- [ ] End-to-end test to cover accelerating only String, Number, Date and Binary [Core Connector Data Types](../definitions.md)
+- [ ] End-to-end test to cover accelerating TPC-H scale factor 1 data from S3 and benchmarking TPC-H queries (official and simple).
+- [ ] The accelerator in all [Access Modes](../definitions.md) successfully loads TPC-H derived data at scale factor 1.
+- [ ] The accelerator in all [Access Modes](../definitions.md) successfully executes at least 75% of TPC-H derived queries at scale factor 1.
+
+### Documentation
+
+- [ ] Documentation includes all information and steps for a user to set up the Accelerator
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]:
```suggestion
- [ ] Documentation includes all information and steps for a user to set up the Accelerator.
```

File: docs/criteria/accelerators/alpha.md [hunk 6]
```diff
@@ -15,4 +15,30 @@ All criteria must be met for the accelerator to be considered Alpha, with except
...
+
+- [ ] The accelerator implements the querying functionality of the native accelerator source in all [Access Modes](../definitions.md)
+- [ ] The accelerator supports only String, Number, Date and Binary [Core Connector Data Types](../definitions.md).
+- [ ] The accelerator executes common use case queries with a low error rate (e.g. `SELECT * FROM tbl WHERE col = 'blah'`).
+- [ ] Known [Minor and Major](../definitions.md) bugs are logged, but not required to be fixed unless needed to achieve a low query error rate and/or TPC-H success.
+  - A "low error rate" indicates that more than 90% of the time, the common use case succeeds.
+
+#### Testing
+
+- [ ] End-to-end test to cover accelerating only String, Number, Date and Binary [Core Connector Data Types](../definitions.md)
```
Comment (phillipleblanc) [inline docs/criteria/accelerators/alpha.md]:
```suggestion
- [ ] End-to-end test to cover accelerating only String, Number, Date and Binary [Core Connector Data Types](../definitions.md).
```

File: docs/criteria/accelerators/alpha.md (hunk 4 above)
Comment (peasee) [reply docs/criteria/accelerators/alpha.md]: > It's not clear to me why we would have any error rate for a given query - either it works or doesn't. Is this a load related metric or something else?

It probably doesn't apply to Accelerators. I included it for Connectors because the set of common use cases can vary by connector, but for accelerators the common use cases are more tightly scoped (and more importantly, usually the same across accelerators). I've removed this item and add an additional item to common use cases to be more tightly scoped.

**Review by phillipleblanc — APPROVED** [review]

**Review by sgrebnov — APPROVED** [review]
